tests/integration/prometheus-tester/config.yaml

```diff
@@ -6,3 +6,7 @@ options:
     default: 1s
     description: "Prometheus job configuration for scrape interval"
     type: string
+  alert-rules-path:
+    default: src/prometheus_alert_rules
+    description: "Path for alert rules passed to the Provider"
+    type: string
```

Comment:
I'm not sure why we are making this configurable. Isn't the correct path already set on charm compile time? Or is this so we can point at different directories all of which are packed in to the charm?

Reply:
It's so we can point at different directories packed in the charm instead of rebuilding the whole thing every time (which causes a mess with `charmcraft` caching)